modules/replicator/README.md

```diff
@@ -0,0 +1,4 @@
+# Ignite replicator module.
```

Comment:
Let's create separate ticket for module description, current one is a bit poor.

Reply:
I created the ticket and added TODO here.